membership-common/src/test/scala/com/gu/memsub/subsv2/services/SubscriptionServiceTest.scala

```diff
@@ -412,14 +359,14 @@ class SubscriptionServiceTest extends Specification {
 
     "Leverage the soap client to fetch subs by contact ID" in {
       val subs = service.current[SubscriptionPlan.AnyPlan](contact)
-      subs.headOption.map(_.name) mustEqual Some(memsub.Subscription.Name("subscriptionNumber")) // what is in the config file
+      subs.map(_.name.get).sorted mustEqual List("A-S00890520", "A-S00890521") // from the test resources jsons
```

Comment:
not sure what the point of testing read the same sub twice, so I read both the annual digi and the monthly S+ and chec they both appear